src/Symfony/Bundle/FrameworkBundle/CHANGELOG.md

```diff
@@ -4,7 +4,7 @@ CHANGELOG
 4.3.0
 -----
 
- * Deprecated the `framework.templating` option, use Twig instead.
+ * Deprecated the `framework.templating` option, configure the Twig bundle instead.
```

Comment:
Should I make this `Twig Bundle`? In the deprecation message for the config I've written it as `Bundle` instead of `bundle`